setup.py

```diff
@@ -29,25 +28,18 @@
             "Environment :: Web Environment",
             "Intended Audience :: Developers",
             "Framework :: Django",
-            "Framework :: Django :: 1.11",
-            "Framework :: Django :: 2.0",
-            "Framework :: Django :: 2.1",
             "Framework :: Django :: 2.2",
             "Framework :: Django :: 3.0",
             "Framework :: Django :: 3.1",
             "Programming Language :: Python",
-            "Programming Language :: Python :: 2.7",
-            "Programming Language :: Python :: 3",
-            "Programming Language :: Python :: 3.4",
             "Programming Language :: Python :: 3.5",
             "Programming Language :: Python :: 3.6",
             "Programming Language :: Python :: 3.7",
             "Programming Language :: Python :: 3.8",
             "License :: OSI Approved :: BSD License",
         ],
-        python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+        python_requires=">=3.5",
```

Comment:
3.5 is already at EOL; perhaps we should support a minimum of 3.6?

Reply:
Django 2.2 is a lts and supports python 3.5. Therefore I think 3.5 should be dropped with Django 2.2.